FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -352,7 +352,7 @@ paths:
       tags:
       - Components
       summary: Post an update on all component properties
-      description: This information allows FRAIHMWORK to track basic information about a component. Any extra or custom fields are expected to always be present even if null values are provided at first. Using this endpoint means that the component needs to be registered (under the hood), so a UUID will be created and assigned in the response
+      description: This information allows FRAIHMWORK to track basic information about a component. Any extra or custom fields are expected to always be present even if null values are provided at first. Using this endpoint means that the component needs to be registered (under the hood), so a UUID will be created and assigned in the response. Note, requests with the fraihmwork.component.write permission may be limited for authorization reasons. See the descriptions in the securitySchemes.OAuth2.flows.clientCredentials.scopes for more details.  
```

Comment:
Actually, this is the one action where I don't think there are any limitations on the write vs admin. Creating a new component should always work for both of these scopes, barring any bad requests